### DISCUSSION TITLE
hyprland vs hyprland-git

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#937] [General]
I guess hyperdots uses the older hyprland and not the git version is that recommended or can I move to hyprland-git?

**Comment by kRHYME7** [comment]
Yes, as long as you can manage the frequent changes. It will also help to know changes in advance. But if you have issues most likely more of the users here won't experience it yet.

**Comment by se7en-x230** [comment]
![240222_11h49m25s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/69777713/f7f75158-ae16-44bc-945c-22131dbec35a)


just to let you know ... 
line 1 and 5 are comments ???

line 85 is the part 
device:epic mouse V1 {
    # sensitivity = -0.5
}

disabled it an now I am fine.


If you want me to report things like that in the future please let me know.

**Reply by kRHYME7** [reply]
Yess thank you! #930  



To correct this here's the syntax



```
device {
name = epic mouse V1 
# sensitivity = -0.5
}
```


>  line 1 and 5 are comments ???

Hyprland added another dependency making "##"  interpreted as something

so to write
````
 ###################
```
it should be 

```
# ##############
```

**Reply by se7en-x230** [reply]
I corrected it already ... thought the double # must be the problem
